Examples/Io/Obj/include/ActsExamples/Io/Obj/ObjTrackingGeometryWriter.hpp

```diff
@@ -41,13 +41,15 @@ class ObjTrackingGeometryWriter {
    public:
     double outputScalor = 1.0;        ///< scale output values
     std::size_t outputPrecision = 6;  ///< floating point precision
-    std::string outputDir = ".";
+    std::filesystem::path outputDir = ".";
```

Comment:
We probably should do this consistently through the examples, it's pretty mix and go.

Reply:
Yes, that would be helpful. The benefit is that this automatically converts from `pathlib.Path` in python.